src/libdrakvuf/libdrakvuf.h

```diff
@@ -223,6 +223,34 @@ struct drakvuf_trap {
     void *data;
 };
 
+
+////////////////////////////////////////////////////////////////////////////
+
+// IMHO these definitions must be placed within another file, named
+// libdrakvuf-windows.h or something similar
+
+// For get_previous_mode...
+typedef enum _MODE {
+    KernelMode,
+    UserMode,
+    MaximumMode
+} MODE ;
+
+// Confirmed only on Win7 SP1...
+typedef enum _OBJECT_MANAGER_OBJECT_TYPE {
+    ObjManagerProcessObject = 7,
```

Comment:
Generally for enum members I like to name them in all capital instead of camel-casing as to be able to differentiate them from variables. So these should be instead OBJ_MANAGER_PROCESS_OBJECT, etc..